connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -106,13 +107,14 @@ message Unknown {}
 
 // Common metadata of all relations.
 message RelationCommon {
-  // TODO(SPARK-48639): Add origin like Expression.ExpressionCommon
-
   // (Required) Shared relation metadata.
-  string source_info = 1;
+  string source_info = 1 [deprecated=true];
```

Comment:
Just switched to deprecation. It breaks the protobuf compat check, and I should backport this change all down to branch-3.4, which isn't worthwhile

Reply:
Can we try annotate it with a buf comment?

Something like 

// buf:lint:ignore FIELD_NO_DELETE